[ADD] Quality: Instructions quality check (#5689)
odoo/documentation: content/applications/inventory_and_mrp/quality/quality_check_types/instructions_check.rst

For a full overview of how to configure a quality check or a |QCP|, see the documentation on
:ref:`Quality checks <quality/quality_management/quality-checks>` and :ref:`Quality Control
Points <quality/quality_management/quality-control-points>`.

Process an Instructions quality check
=====================================

There are multiple ways that Instructions quality checks can be processed. If a quality check is
assigned to a specific manufacturing, inventory, or work order, the check can be processed on the
order itself. Alternatively, a check can be processed from the check's page.

Process from the quality check's page
-------------------------------------

To process an Instructions quality check from the check's page, begin by navigating to
:menuselection:`Quality --> Quality Control --> Quality Checks`, and select a quality check. Follow
the :guilabel:`Instructions` for how to complete the check.

If the product passes the check, click the :guilabel:`Pass` button above the quality check form. If
the product does not pass the check, click the :guilabel:`Fail` button, instead.

Process a quality check on an order
-----------------------------------

> To process an Instructions quality check on an order, select a manufacturing order or inventory

Reviewer: > inventory order 

I don't think we have a definitive way of naming things, but I typically see the BSAs calling these "warehouse transfers" or "inventory transfers". I don't have a super strong opinion about "inventory order" but it might be confusing when linked to a sales order. 

If you do decide to change "inventory orders" to warehouse transfers, I suggest that you also change the content within the parantheses to "receipt, delivery order, return, etc."

order (receipt, delivery, return, etc.) for which a check is required. Manufacturing orders can be
selected by navigating to :menuselection:`Manufacturing --> Operations --> Manufacturing Orders`,
and clicking on an order. Inventory orders can be selected by navigating to

> :menuselection:`Inventory`, clicking the :guilabel:`# To Process` button on an operation card, and
> selecting an order.

Reviewer: ```suggestion
:menuselection:`Inventory`, clicking the :guilabel:`# To Process` button on an operation type card,
and selecting an order.
```

On the selected manufacturing or inventory order, a purple :guilabel:`Quality Checks` button appears
above the order. Click the button to open the :guilabel:`Quality Check` pop-up window, from which
any quality checks created for the order can be processed.

.. image:: instructions_check/quality-check-pop-up.png
   :align: center
   :alt: The Quality Check pop-up window on a manufacturing or inventory order.

To complete an Instructions quality check, follow the instructions detailed in the
:guilabel:`Quality Check` pop-up window. Finally, click :guilabel:`Validate` to confirm that the
check has been completed.

> If a quality alert must be created, click the :guilabel:`Quality Alert` button that appears at the

Reviewer: As someone not super familiar with Quality, it's not obvious why a quality alert is raised at this step. Maybe starting this paragraph with a little bit more context on when Quality Alerts are raised, it would be helpful (::

```suggestion
If the quality check failed, raise a quality alert to ____. To do so, click the :guilabel:`Quality Alert` button that appears at the...
```
(I left the explanation empty because i dont know :joy: )

When configuring a |QCP| that is triggered by a manufacturing order, a specific work order can also
be specified in the :guilabel:`Work Order Operation` field on the |QCP| form. If a work order is
specified, an Instructions quality check is created for that specific work order, rather than the
manufacturing order as a whole.

Quality checks configured for work orders must be completed from the tablet view. To do so, begin by
navigating to :menuselection:`Manufacturing --> Operations --> Manufacturing Orders`. Select a
manufacturing order that includes a work order for which a quality check is required. Open the
tablet view for that work order by clicking the :guilabel:`📱 (tablet)` button on the order's line.

With tablet view open, complete the steps listed on the left side of the screen until the
Instructions quality check step is reached. Upon reaching the check, the instructions for how to
complete it will appear at the top of the screen. Follow the instructions, then click
:guilabel:`Next` to move on to the next step.

.. image:: instructions_check/work-order-instructions-check.png
   :align: center
   :alt: An instructions check for a work order.

> If a quality alert must be created, do so by clicking the :guilabel:`☰ (menu)` button in the tablet
> view, and selecting :guilabel:`Quality Alert` from the :guilabel:`Menu` pop-up window. A
> :guilabel:`Quality Alerts` pop-up window appears, from which a quality alert can be created. For a
> complete guide to quality alert creation, view the documentation on :ref:`Quality alerts
> <quality/quality_management/quality-alerts>`.

Reviewer: If I remember correctly, the () after the emoji is to describe what the emoji looks like, in case the reader's browser is too old to render the emoji.
```suggestion
If a quality alert must be created, do so by clicking the :guilabel:`☰ (hamburger)` icon in the top
left corner tablet view, and selecting :guilabel:`Quality Alert` from the :guilabel:`Menu` pop-up
window.

Then, a :guilabel:`Quality Alerts` pop-up window appears, from which a quality alert can be
created. For a complete guide to quality alert creation, view the documentation on :ref:`Quality
alerts <quality/quality_management/quality-alerts>`.
```

==========================
Instructions quality check
==========================

.. |QCP| replace:: :abbr:`QCP (Quality Control Point)`

In Odoo *Quality*, an *Instructions* check is one of the quality check types that can be selected
when creating a new quality check or quality control point (QCP). Instructions checks consist of a
text entry field that allows the creator to provide instructions for how to complete the check.

For a full overview of how to configure a quality check or a |QCP|, see the documentation on
:ref:`quality checks <quality/quality_management/quality-checks>` and :ref:`quality control
points <quality/quality_management/quality-control-points>`.

Process an Instructions quality check
=====================================

> There are multiple ways that Instructions quality checks can be processed. If a quality check is

Reviewer 2: ```suggestion
There are multiple ways that *Instructions* quality checks can be processed. If a quality check is
```

assigned to a specific manufacturing, inventory, or work order, the check can be processed on the
order itself. Alternatively, a check can be processed from the check's page.

Process from the quality check's page
-------------------------------------

> To process an Instructions quality check from the check's page, begin by navigating to

Reviewer 2: we're indirectly referring to an Odoo UI label in these contexts so let's italicize otherwise the capitalization (+ other formatting) isn't needed
```suggestion
To process an *Instructions* quality check from the check's page, begin by navigating to
```
